x-pack/legacy/plugins/monitoring/server/plugin.js

```diff
@@ -72,7 +72,11 @@ export class Plugin {
       reserved: {
         privilege: {
           savedObject: {
-            all: [],
+            all: [
+              'alert',
+              'action',
+              'action_task_params',
+            ],
             read: []
           },
           ui: [],
```

Comment:
I'm noticing the structure for reserved privileges (or at least this one) is different than usual. Is there a way to set `ui: []` differently between `all` and `read` on reserved privileges?